Documentation/Visualizations/Grid.md

```diff
@@ -86,22 +86,31 @@ Here is an example that styles the _Request_ column as a bar:
 ### Link Actions
 If the `Link` renderer is selected, or the `Make this item a link` checkbox is selected, then the author can  configure a link action that will occur on clicking the cell. This usually is taking the user to some other view with context coming from the cell, or may open up a url.
 
-The list of link actions include:
-
+#### General Link Actions
 | Link action | Action on click |
 |:------------- |:-------------|
 | `Generic Details` | Shows the row values in a property grid context blade |
 | `Cell Details` | Shows the cell value in a property grid context blade. Useful when the cell contains a dynamic type with information (e.g. json with request properties like location, role instance, etc.). |
-| `Cell Details` | Shows the cell value in a property grid context blade. Useful when the cell contains a dynamic type with information (e.g. json with request properties like location, role instance, etc.). |
+| `Url` | The value of the cell is expected to be a valid http url, and the cell will be a link that opens up that url in a new tab.|
+
+#### Application Insights
+| Link action | Action on click |
+|:------------- |:-------------|
 | `Custom Event Details` | Opens the Application Insights search details with the custom event id (itemId) in the cell |
 | `* Details` | Similar to Custom Event Details, except for dependencies, exceptions, page views, requests and traces. |
 | `Custom Event User Flows` | Opens the Application Insights User Flows experience pivoted on the custom event name in the cell |
 | `* User Flows` | Similar to Custom Event User Flows except for exceptions, page views and requests |
```

Comment:
Not sure if you know why, maybe ask John if not sure, but why does Details and User Flows have * ? There's nothing after this section that explain what * means